diesel/src/types/impls/date_and_time/chrono.rs

```diff
@@ -100,4 +124,53 @@ mod tests {
             .unwrap().nth(0).unwrap()
         );
     }
+
+    #[test]
+    fn times_of_day_encode_correctly() {
+        let connection = connection();
+
+        let midnight = NaiveTime::from_hms(0, 0, 0);
+        assert!(
```

Comment:
I'm about to push up a fix for #97, and would like this to be updated to use it before I merge as I'm going to remove `query_sql_params`. The code for this will be:

```rust
let connection = connection();
let midnight = NaiveTime::from_hms(0, 0, 0);
let query = select(sql::<Time>("'00:00:00'").eq(midnight));
assert!(query.get_result::<bool>(&connection).unwrap());
```

I'll ping you when it's up, probably 5-10 more minutes.